### PR TITLE
add dependency map to manifest

### PR DESCRIPTION
**Pull request by OsmondJiang**

### CODE REVIEW

**Comment by OsmondJiang** [comment]
#2816

File: docs/designs/output.md [hunk 1]
```diff
@@ -126,9 +128,9 @@ These are called *browser navigation properties* and are stored in manifest.
 {
     "files": [{
         "url": "/dotnet/api/system.string",
-        "path": "en-us/netstandard-2.0/dotnet/api/system.string.json",
+        "site-path": "en-us/netstandard-2.0/dotnet/api/system.string.json",
         "locale": "en-us",
-        "moniker": "netstandard-2.0",
+        "moniker": "netstandard-2.0", 
 
         // other browser navigation properties...
```
Comment (yufeih) [inline docs/designs/output.md]: How about:
```json
"files": [{
    "source": "system.string.yml",
    "siteUrl": "/dotnet/api/system.string",
    "outputPath": "en-us/netstandard-2.0/dotnet/api/system.string.json",
    "locale": "en-us",
    "moniker": "netstandard-2.0",
    "dependencies": [{
        "type": "link",
        "source": "a.md"
    }]
}]
```

Comment (OsmondJiang) [reply docs/designs/output.md]: it would be a little big,  for difference purpose, i suggest to put them in two sections

Comment (OsmondJiang) [reply docs/designs/output.md]: and dependencies has no relationship with moniker?

**Pull request merged**